src/test/scala/chiseltest/tests/SingleElaboration.scala

```diff
@@ -0,0 +1,50 @@
+// See LICENSE for license details.
+
+package chiseltest.tests
+
+import chisel3._
+import chiseltest._
+import chiseltest.experimental.TestOptionBuilder._
+import chiseltest.internal.VerilatorBackendAnnotation
+import firrtl.AnnotationSeq
+import org.scalatest._
+
+class SingleElaborationTests extends FlatSpec with ChiselScalatestTester with Matchers {
+
+  behavior of "elaboration should happen once"
+
+  class ThingAMaBobBundle extends Bundle {
+    val condition = Input(Bool())
+  }
+
+  class ThingAMaBob extends Module {
```

Comment:
Is this module critical to test functionality, or can you use one of the standard modules in TestUtils (and maybe we should also have an empty DummyModule there too, since I think the module contents are orthogonal to this test case)?

Reply:
I created a DummyModule